TasksAboutArrays/ArraySizeWithN.java

```diff
@@ -0,0 +1,37 @@
+package TasksAboutArrays;
+import java.util.Scanner;
+public class ArraySizeWithN {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int n = scanner.nextInt();
+
+        if(n == 0 || n < 0){
+            System.out.println("n should be natural number.");
+            System.exit(2);
+        }
+
+        int[] array;
+        switch (n) {
+            case 3:
+                array = new int[3];
+                break;
+            case 4:
+                array = new int[4];
+                break;
+            case 5:
+                array = new int[5];
+                break;
```

Comment:
This can be shortened, if you think a bit longer)

Reply:
just switch (n) {
            case 3:
            case 4:
            case 5:
                array = new int[n];
                break;